### ISSUE TITLE
retest this please

### ISSUE THREAD

**Issue by PavelAgejkin** [ashwanthkumar/gocd-build-github-pull-requests#60]
Hi,
I cant find "retest this please" feature.
Is there any options to trigger go pipeline again, without commit?

**Comment by ashwanthkumar** [comment]
Have you tried "Trigger with options"? 

PS: Personally I've not tried it.

**Comment by PavelAgejkin** [comment]
Could you please point, where I can find "Trigger with options"?

**Comment by ashwanthkumar** [comment]
TLDR - It is the Play with button with a + on the pipeline.

Ref -
https://www.go.cd/documentation/user/current/advanced_usage/trigger_with_options.html


On Sun, Dec 13, 2015 at 9:04 PM, Pavel <notifications@github.com> wrote:

> Could you please point, where I can find "Trigger with options"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ashwanthkumar/gocd-build-github-pull-requests/issues/60#issuecomment-164268315>
> .
>



-- 

Ashwanth Kumar / ashwanthkumar.in

**Comment by ashwanthkumar** [comment]
@PavelAgejkin I just saw the comment was edited. Do you mean the notification plugin integration? Simple answer - Yes. It runs for every instance of a pipeline / job. Specifically for Stash may be @srinivasupadhya can answer. I haven't used Stash, sorry.

**Comment by virtuald** [comment]
The Jenkins plugin has an option to retrigger a build based on the contents of comments in the PR. It's a bit more intuitive then having to use GoCD's mechanism to do it -- and less steps. 

"Computer, test this please"

**Comment by wlsf82** [comment]
Yeah, I really think GoCD could have something similar to this Jenkins plugin, then there is no need to access GoCD, you can trigger the pipeline again directly from Github.

**Comment by ashwanthkumar** [comment]
It would be great if someone can send a PR for this feature, or if someone is interested in picking this up - I can help too.

**Comment by abhijaju** [comment]
Any update on this ?

**Comment by ashwanthkumar** [comment]
I'm sorry, I don't have an update on this :disappointed: It would be great if someone can send a PR which I would be glad to review and merge it in.

**Comment by wlsf82** [comment]
I'm already used to not having this.
